priv/repo/new_venues.exs

```diff
@@ -198,10 +167,94 @@ defmodule CsGuide.NewVenues do
 
     Regex.split(postcode_regex, str, include_captures: true, trim: true)
   end
+
+  defp add_drinks_to_map(map, drinks) do
+    Map.put(map,
+      :drinks,
+      Map.new(
+        Enum.map(
+          drinks,
+          fn {d, b} ->
+            brand = Brand.get_by(name: b)
+
+            case Drink.get_by(name: d, brand_id: brand.id) do
+              nil ->
+                IO.inspect("Drink not found: #{d}, brand: #{b}")
+                nil
+
+              drink ->
+                {drink.entry_id, "on"}
+            end
+          end
+        )
+        |> Enum.filter(fn d -> not is_nil(d) end)
+      )
+    )
+  end
+
+  defp add_users_to_map(map, venue) do
+    if Map.has_key?(venue, :email) && venue.email != "" do
+      Map.put(map, :users, %{"0" => %{"email" => venue.email, "role" => "venue_admin"}})
+    else
+      map
+    end
+  end
+
+  # some venues don't have a phone_number so don't attempt to String.replace!
+  defp sanitise_phone_number(map, venue) do
+    # IO.inspect(map, label: "map")
```

Comment:
Are these inspects needed?

Reply:
can be removed. I commented out in case they were useful for the next import.